docs/pages/docs/examples.tsx

```diff
@@ -248,6 +248,16 @@ export default function Docs() {
         >
           Example to demonstrate customisation of Keystone's document field and document renderer.
         </Well>
+
+        <Well
+          grad="grad2"
+          heading="Next.js + Keystone"
+          href="https://github.com/keystonejs/keystone/blob/main/examples/nextjs-keystone"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Shows you how to use Keystone as a data engine withing Next.js applications.
```

Comment:
```suggestion
          Shows you how to use Keystone as a data engine within Next.js applications.
```

Reply:
thanks for catching this. I've pushed this change along with another change in a separate commit.